index.html

```diff
@@ -215,22 +215,22 @@ <h2>Contact me</h2>
       that you need built or a project that needs coding. I'd love to help
       with it
     </p>
-    <form action="https://formspree.io/f/xbjwwqjp" method="post">
-      <ul class="contact-ul">
-        <li>
-          <input type="text" class="input" name="name" maxlength="30" placeholder="Enter your name" />
-        </li>
-        <li>
-          <input type="email" class="input" placeholder="Enter your email" name="email" required />
-        </li>
-        <li>
-          <textarea class="text-area" name="message" maxlength="500" placeholder="Write your message here"
-            required></textarea>
-        </li>
-        <li>
-          <button type="submit">Get in Touch</button>
-        </li>
-      </ul>
+    <form action="https://formspree.io/f/xeqndwyy" id="form" method="post">
+
+      <div class="contact-form">
+        <input id="name" type="text" class="input" name="name" maxlength="30" placeholder="Enter your name" /><br>
+        <span class="error"></span>
+      </div>
+      <div class="contact-form">
+        <input id="email" type="email" class="input" name="email" maxlength="30" placeholder="Enter your email" /><br>
+        <span class="error"></span>
+      </div>
+      <div class="contact-form">
+        <textarea class="text-area" id="message" name="message" maxlength="500"
+          placeholder="Write your message here"></textarea><br>
+        <span class="error"></span>
+      </div>
+      <button type="submit">Get in Touch</button>
```

Comment:
- [ ] As stated in the project requirements: you should show an error message to the user near the submit button informing them of the error and that the form is not sent. In your case, the message is displayed under the input. Please remove it and make sure the message shows up close to the button `submit`.
![image](https://user-images.githubusercontent.com/80612925/174115601-1174d9e0-78e9-4809-9c94-31b612972c08.png)

![image](https://user-images.githubusercontent.com/80612925/174115441-5e94a7fc-abca-4430-8864-4798882ad165.png)